src/mmw/apps/modeling/views.py

```diff
@@ -526,7 +532,90 @@ def boundary_layer_detail(request, table_code, obj_id):
             geojson = json.loads(GEOSGeometry(row[0]).geojson)
             return Response(geojson)
         else:
-            return Response(status=status.HTTP_400_BAD_REQUEST)
+            return Response(status=status.HTTP_404_NOT_FOUND)
+
+
+def _get_boundary_search_query(search_term):
+    """
+    Return raw SQL query to perform full text search against
+    all searchable boundary layers.
+    """
+    select_fmt = """
+        SELECT
+            id,
+            '{code}' AS code,
+            name,
+            {priority} AS priority,
+            ST_Centroid(ST_Transform(geom, 4326)) as center
```

Comment:
Oh, I misread this comment. I'm going to fix this by removing `ST_Transform` which was a little ceremonious since these boundaries should be imported as 4326 anyway.